Update manage-notification-methods.md (#19601)
netdata/netdata: docs/alerts-and-notifications/notifications/centralized-cloud-notifications/manage-notification-methods.md

2. Click on the **Alerts & Notifications** tab on the left-hand side.
3. Click on the **Notification Methods** tab.
4. You will be presented with a table of the configured notification methods for the Space. You will be able to:
    - **Add a new** notification method configuration.
        - Choose the service from the list of available ones. The available options will depend on your subscription plan.
        - You can optionally provide a name for the configuration so you can refer to it.

>         - You can define the filtering criteria, regarding which Rooms the method will apply, and what notifications you want to receive (All Alerts and unreachable, All Alerts, Critical only).

Reviewer: This one should also have the same options as the notification selection / filtering with the personal (email/ mobile app) notification settings.

>         - You can define the filtering criteria, regarding which Rooms the method will apply, and what notifications you want to receive (All Alerts and unreachable, All Alerts, Critical only).

Author: ```suggestion
        - You can define the filtering criteria, regarding which Rooms the method will apply, and what notifications you want to receive (Critical, Warning, Clear, Reachable and Unreachable).
```

To manage user-specific notification settings, you will need the following:

- A Cloud account

> - Access to, at least, a Space

Reviewer 2: This is an interesting case! Usually, we see missing commas, but here we have the opposite problem

> **Note**
>
> If an administrator has disabled a Personal [service level](/docs/alerts-and-notifications/notifications/centralized-cloud-notifications/centralized-cloud-notifications-reference.md#service-level) notification method, this will override any user-specific setting.

### Steps

1. Click on your profile picture and navigate to **Settings** -> **Notifications**
2. You are presented with:
    - The Personal [service level](/docs/alerts-and-notifications/notifications/centralized-cloud-notifications/centralized-cloud-notifications-reference.md#service-level) notification methods you can manage.
    - The list of Spaces and Rooms you have access to.
3. On this modal you will be able to:
    - **Enable/Disable** the notification method on a personal scope, this applies across all Spaces and Rooms.

>     - **Define what notifications you want to receive** per Space/Room: Critical, Warning, Clear, Unreachable.

Author: ```suggestion
    - **Define what notifications you want to receive** per Space/Room: Critical, Warning, Clear, Reachable and Unreachable.
```
